internal/cmd/base/create.go

```diff
@@ -0,0 +1,79 @@
+package base
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/hetznercloud/cli/internal/cmd/output"
+	"github.com/hetznercloud/cli/internal/cmd/util"
+	"github.com/hetznercloud/cli/internal/hcapi2"
+	"github.com/hetznercloud/cli/internal/state"
+	"github.com/hetznercloud/hcloud-go/v2/hcloud"
+)
+
+// CreateCmd allows defining commands for resource creation
+type CreateCmd struct {
+	BaseCobraCommand func(hcapi2.Client) *cobra.Command
+	Run              func(context.Context, hcapi2.Client, state.ActionWaiter, *cobra.Command, []string) (*hcloud.Response, any, error)
+	PrintResource    func(context.Context, hcapi2.Client, *cobra.Command, any)
+}
+
+// CobraCommand creates a command that can be registered with cobra.
+func (cc *CreateCmd) CobraCommand(
+	ctx context.Context, client hcapi2.Client, tokenEnsurer state.TokenEnsurer, actionWaiter state.ActionWaiter,
+) *cobra.Command {
+	cmd := cc.BaseCobraCommand(client)
+
+	output.AddFlag(cmd, output.OptionJSON())
```

Comment:
Could you also add it to the docs: https://github.com/hetznercloud/cli?tab=readme-ov-file#output-configuration

Reply:
Done in 8e06561252d1ac6306abb778fd542db32b243f9b